### PR TITLE
Cleanup and move cfg docs

### PR DESCRIPTION
**Pull request by ondrejtomcik**

### CODE REVIEW

File: content/en/architecture/component-overview.md [hunk 1]
```diff
@@ -405,49 +405,6 @@ plgd cloud uses [NATS](https://nats.io) messaging system as it's event bus.
 
 ## Event Store
 
-plgd cloud persist events in an event store, which is a database of events. The store has an API for adding and retrieving device's events. Events needs to be versioned and written in a correct order. To achieve the consistency, optimistic concurrency control method is applied during each write.
-After the event is successfully written into the event store, it's distributed to the event bus which notifies all subscribers about the change asynchronously.
+The plgd Cloud stores each resource state mutation as a separate record called event. This enables us to reconstructs past states, use events as a foundation to understand user's behaviour or even explore an tamper-proof audit log. The plgd Cloud adopts [Event Sourcing](https://docs.microsoft.com/en-us/azure/architecture/patterns/event-sourcing) and [CQRS](https://docs.microsoft.com/en-us/azure/architecture/patterns/cqrs) patterns to support flexibility, scalability and trackability of the system.
```
Comment (Danielius1922) [inline content/en/architecture/component-overview.md:408]: reconstructs -> reconstruct
an tamper-proof -> a tamper-proof

(I use https://languagetool.org/ to check grammar - articles, commas, etc.)

File: content/en/configuration/cloud2cloud-gateway.md [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+categories: [configuration, deployment]
+keywords: [configuration]
+menu:
+  docs:
+    parent: configuration
+    weight: 70
+toc: true
+---
+
+C2C Gateway implements [Open Connectivity Foundation's Cloud API for Cloud Services](https://openconnectivity.org/developer/specifications/) providing API's to mirror user's devices to a different OCF standardized cloud.
```
Comment (jkralik) [inline content/en/configuration/cloud2cloud-gateway.md:15]: point `Open Connectivity Foundation's Cloud API for Cloud Services` to right doc

File: content/en/configuration/coap-gateway.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+#### OAuth2.0 Client
+
+OAuth2.0 Client is used to issuing an authorization code used by the Onboarding Tool during the device provisioning which is then exchanged for an access token during the [SignUp operation](https://plgd.dev/architecture/component-overview/#cloud-registration).
+
+| Property | Type | Description | Default |
+| ---------- | -------- | -------------- | ------- |
+| `api.coap.authorization.ownerClaim` | string | `Claim used to identify owner of the device.` | `"sub"` |
+| `api.coap.authorization.deviceIdClaim` | string | `Claim used to identify device id of the device. Empty means that JWT doesn't contain it.` | `""` |
+| `api.coap.authorization.providers[].Name` | string | `Provider name, the register request of the device must contain one of the names of providers.` | `""` |
```
Comment (jkralik) [inline content/en/configuration/coap-gateway.md:63]: `Name` ??? right is `name`

File: content/en/quickstart/device-to-cloud.md [hunk 4]
```diff
@@ -14,4 +14,6 @@ toc: true
 
 ## try.plgd.cloud
 
-The plgd team operates their own instance of the plgd cloud for free. This cloud instance is integrated with the plgd mobile application available for both iOS and Android based devices. Together with our IoTivity-Lite sample you're able to onboard and work with your device remotely in couple of seconds. To start right away, follow [try.plgd.cloud](https://try.plgd.cloud).
+To try the plgd Cloud without downloading a single docker container, [try.plgd.cloud](https://try.plgd.cloud) instance is available for you for free. To onboard any IoTivitiy-Lite or any OCF standardized device to the plgd Cloud, the mobile application **plgd** is available for you in both, [Apple](https://apps.apple.com/sk/app/plgd/id1536315811) and [Google Play](https://play.google.com/store/apps/details?id=dev.plgd.client&hl=sk&gl=US) stores.
```
Comment (jkralik) [inline content/en/quickstart/device-to-cloud.md:17]: pls the change link to english: 
`https://play.google.com/store/apps/details?id=dev.plgd.client&hl=en&gl=US`

File: content/en/configuration/identity-server.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
+---
+title: 'Identity Server'
```
Comment (jkralik) [inline content/en/configuration/identity-server.md:2]: rename all `Identity Server` to `Identity Service`

File: content/en/configuration/cloud2cloud-gateway.md (hunk 2 above)
Comment (ondrejtomcik) [reply content/en/configuration/cloud2cloud-gateway.md:15]: Reason why I pointed to /specification is that your link points to v2.2.4. And we don't want to update documentation every time OCF releases a new spec.

File: content/en/configuration/coap-gateway.md (hunk 3 above)
Comment (ondrejtomcik) [reply content/en/configuration/coap-gateway.md:63]: I just moved the file from cloud repo. Will change it to name.

File: content/en/architecture/component-overview.md (hunk 1 above)
Comment (ondrejtomcik) [reply content/en/architecture/component-overview.md:408]: Thanks 👍🏽

File: content/en/quickstart/device-to-cloud.md (hunk 4 above)
Comment (ondrejtomcik) [reply content/en/quickstart/device-to-cloud.md:17]: done

File: content/en/configuration/identity-server.md (hunk 5 above)
Comment (ondrejtomcik) [reply content/en/configuration/identity-server.md:2]: To Identity or to Identity Service :D

File: content/en/configuration/cloud2cloud-gateway.md (hunk 2 above)
Comment (jkralik) [reply content/en/configuration/cloud2cloud-gateway.md:15]: Then you can point to https://openconnectivity.org/specs/OCF_Cloud_API_For_Cloud_Services_Specification.pdf

File: content/en/configuration/identity-server.md (hunk 5 above)
Comment (jkralik) [reply content/en/configuration/identity-server.md:2]: to identity store :)

File: content/en/configuration/cloud2cloud-gateway.md (hunk 2 above)
Comment (ondrejtomcik) [reply content/en/configuration/cloud2cloud-gateway.md:15]: Done

Comment (jkralik) [reply content/en/configuration/cloud2cloud-gateway.md:15]: I dont see any update.

File: content/en/configuration/identity-server.md (hunk 5 above)
Comment (ondrejtomcik) [reply content/en/configuration/identity-server.md:2]: done

File: content/en/architecture/component-overview.md [hunk 6]
```diff
@@ -248,25 +248,25 @@ hide footbox
 participant C as "Client"
 participant GGW as "gRPC Gateway"
 participant RA as "Resource Aggregate"
-participant AS as "Authorization Server"
+participant IS as "Identity Server"
```
Comment (jkralik) [inline content/en/architecture/component-overview.md:251]: Identity Store

File: content/en/architecture/component-overview.md [hunk 7]
```diff
@@ -99,13 +99,13 @@ hide footbox
 
 participant D as "Device"
 participant CGW as "CoAP Gateway"
-participant AS as "Authorization Server"
+participant IS as "Identity Server"
```
Comment (jkralik) [inline content/en/architecture/component-overview.md:102]: Identity Store

**Review by jkralik — APPROVED** [review]

File: content/en/configuration/resource-directory.md [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+| `clients.identityStore.grpc.keepAlive.time` | string | `After a duration of this time if the client doesn't see any activity it pings the server to see if the transport is still alive.` | `10s` |
+| `clients.identityStore.grpc.keepAlive.timeout` | string | `After having pinged for keepalive check, the client waits for a duration of Timeout and if no activity is seen even after that the connection is closed.` | `20s` |
+| `clients.identityStore.grpc.keepAlive.permitWithoutStream` | bool | `If true, client sends keepalive pings even with no active RPCs. If false, when there are no active RPCs, Time and Timeout will be ignored and no keepalive pings will be sent.` | `false` |
+
+### Public Configuration
+
+These configurations are `Coap Cloud Conf` information for device registration to plgd cloud as well as root CA certificate, certificate authority address to get identity certificate for ssl connection to plgd cloud before device registration.
+This will be served by HTTP Gateway API as defined [here](https://github.com/plgd-dev/cloud/blob/v2/http-gateway/uri/uri.go#L14) and also see [cloud-configuration](https://try.plgd.cloud/.well-known/cloud-configuration).
```
Comment (Danielius1922) [inline content/en/configuration/resource-directory.md:115]: We need to grep this repo for `v2` once we switch to `main` branch.

**Pull request merged**